package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.388.0",
     "@aws-sdk/node-http-handler": "^3.374.0",
-    "@azure/storage-blob": "^12.11.0",
+    "@azure/storage-blob": "12.18.x",
```

Comment:
we are not working with the newer versions and do we know why? should we have a follow up ticket to upgrade to the newer versions?

Reply:
I added a comment and created the jira - SNOW-1475890